api/lagoon/client/_lgraphql/organizations/organizationById.graphql

```diff
@@ -0,0 +1,60 @@
+query (
+  $id: Int!) {
+    organizationById(
```

Comment:
this query is very large and could take a lot of time to return data in some cases, it might be best to split this out into smaller queries

organizationById should just return the information about the organization itself

then have one for `deploytargets` and one for `projects`, `users`, `groups`, `notifications`, etc...

The idea is that you'd never be able to cleanly display the full return of this payload in a response in the CLI

Reply:
Yep haven't done any optimization yet, all still a wip.